hot-reload-analysis/src/main/kotlin/org/jetbrains/compose/reload/analysis/ClassInfo.kt

```diff
@@ -56,3 +59,17 @@ internal fun ClassInfo(classNode: ClassNode): ClassInfo? {
         flags = ClassFlags(classNode.access)
     )
 }
+
+private fun getResourceContentHash(fieldNode: FieldNode?): Int? {
+    try {
+        return fieldNode?.invisibleAnnotations?.find { isResourceContentHashAnnotation(it) }?.values[1] as Int?
+    } catch (_: Throwable) {
+        return null
+    }
+}
+
+fun isResourceContentHashAnnotation(it: AnnotationNode): Boolean =
+    it.desc == "Lorg/jetbrains/compose/resources/ResourceContentHash;"
```

Comment:
Can we use ClassId and define it in the universal `Ids` object?

Reply:
yes, done